.changeset/slimy-dragons-promise.md

```diff
@@ -0,0 +1,20 @@
+---
+"@salt-ds/theme": minor
+---
+
+Added tertiary variables in theme next and existing theme
```

Comment:
Some token only exist in theme next, some are shared, better to split.

Also need to mention un-deprecate

Reply:
done